src/connector/src/source/filesystem/opendal_source/opendal_enumerator.rs

```diff
@@ -51,8 +51,16 @@ impl<Src: OpendalSource> SplitEnumerator for OpendalEnumerator<Src> {
 
     async fn list_splits(&mut self) -> ConnectorResult<Vec<OpendalFsSplit<Src>>> {
         let empty_split: OpendalFsSplit<Src> = OpendalFsSplit::empty_split();
+        let prefix = self.prefix.as_deref().unwrap_or("/");
```

Comment:
Why do we use `""` as the default when prefix is absent in L69 but use `/` here?

Reply:
Actually both is fine, see https://github.com/apache/opendal/blob/690e49467228602cccabe78ba8403fd9123146d4/core/src/raw/path.rs#L96.
For the sake of consistency, I have changed the previous default to `/`.